CIP-high-assurance/README.md

```diff
@@ -0,0 +1,173 @@
+---
+CIP: XXX
+Title: High Assurance Best Practices
+Category: Meta
```

Comment:
```suggestion
Category: Tools
```
This cannot be a `Meta` proposal unless it refers to the CIP process itself... e.g., if it were only to contain best practice standards for CIP implementations.  I'm suggesting `Tools` because the proposal mainly describes off-chain activities.